src/client/datascience/notebook/cssStyles/tableStyles.js

```diff
@@ -0,0 +1,3 @@
+var tableStyleSheet = document.createElement('style');
+tableStyleSheet.innerHTML = "#container > div > div:not(.preview) > div {overflow-x: overlay;}";
```

Comment:
Initially I was trying to tweak table styles here. Instead just applying @rebornix 's fix here early via the preload: https://github.com/microsoft/vscode/commit/d111d967b72748bbd125d3392eeebc52718a7c07

Note: I would only submit to release branch.

Reply:
why not add a css file and import that into one of the ts files.
i personally think that might be simpler than changing webpack and loading more scripts into the controller.
else we end up setting a pattern to create more webpack entries with more js files.

Reply:
could we prabably rename fontawesome loader js into styleloader. js and have that load css files